llvm/lib/Target/RISCV/RISCVSchedSyntacoreSCR345.td

```diff
@@ -0,0 +1,449 @@
+//==- RISCVSchedSyntacoreSCR345.td - Syntacore SCR3, SCR4, SCR5 Scheduling Definitions -*- tablegen -*-=//
```

Comment:
Line exceeds 80 characters

Reply:
Addressed. Also fixed all other long lines in this file.